diff -Naur parprouted-0.42.orig/Makefile parprouted-0.42/Makefile
--- parprouted-0.42.orig/Makefile	2002-09-18 10:14:42.000000000 +0200
+++ parprouted-0.42/Makefile	2003-09-25 16:48:14.000000000 +0200
@@ -1,18 +1,16 @@
 EXTRA_CFLAGS = 
 
 CC = gcc
-
 CFLAGS = -g -O2 -Wall $(EXTRA_CFLAGS)
 LDFLAGS = -s 
 OBJS = parprouted.o arp.o
-
 LIBS = -lpthread
 
 all: parprouted
 
 install: all
-	install parprouted /usr/local/sbin
-	install parprouted.8 /usr/local/man/man8
+	install parprouted ${DESTDIR}/usr/sbin
+	install parprouted.8 ${DESTDIR}/usr/share/man/man8
 
 clean:
 	rm -f $(OBJS) parprouted core
